portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,43 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import com.google.gson.Gson;
+
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Helena!</h1>");
+    //step1
+    //response.setContentType("text/html;");
+    //response.getWriter().println("<p>Hello Helena!</p>");
+
+    //step3
+    ArrayList<String> pod = new ArrayList<String>();
+    pod.add("Neriyah");
+    pod.add("Evie");
+    pod.add("Helena");
+
+    // Convert the server stats to JSON
+    String json = convertToJsonUsingGson(pod);
+
+    // Send the JSON as the response
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+
+  private String convertToJsonUsingGson(ArrayList<String> list) {
```

Comment:
convertToJson() would be sufficient. We rarely add implementation detail in the method name.